<launch>

    <arg name="url" default="rtsp://192.168.1.101:8554/mjpeg/1" />
    <arg name="camera_name" default="http_camera" />
    <arg name="frame" default="http_camera" />
    <arg name="topic" default="/camera/image_raw" />
    <arg name="camera_info_topic" default="camera_info" />
    <arg name="camera_info_url" default="tmp" />
    <arg name="fps" default="10" />
    <arg name="verbose" default="False" />
    <arg name="respawn" default="false" />

    <node name="http_camera" pkg="sibot_video" type="rtsp_cam_publisher.py" respawn="$(arg respawn)" output="screen" >
        <param name="rtsp_resource" value="$(arg url)" />
        <param name="camera_name" value="$(arg camera_name)" />
        <param name="camera_frame" value="$(arg frame)" />
        <param name="image_raw_topic" value="$(arg topic)" />
        <param name="camera_info_topic" value="$(arg camera_info_topic)" />
        <param name="camera_fps" value="$(arg fps)" />
        <param name="verbose" value="$(arg verbose)" />
        <param name="camera_info_url" value="$(arg camera_info_url)" />
    </node>


</launch>